src/components-examples/angular/processflow/processflow-overview/processflow-overview-example.html

```diff
@@ -0,0 +1,48 @@
+<button sbb-secondary-button (click)="linear = !linear" id="toggle-linear">
```

Comment:
I'd prefer to use a checkbox here as we did in many other examples for boolean switching. In addition we normally place configuration parts after the example (here after sbb-processflow tag)